aviary/interface/reports.py

```diff
@@ -42,3 +52,118 @@ def subsystem_report(prob, **kwargs):
 
     for subsystem in core_subsystems.values():
         subsystem.report(prob, reports_folder, **kwargs)
+
+
+def mission_report(prob, **kwargs):
+    """
+    Creates a basic mission summary report that is place in the "reports" folder
+
+    Parameters
+    ----------
+    prob : AviaryProblem
+        The AviaryProblem used to generate this report
+    """
+    def _get_phase_value(traj, phase, var_name, units, indices=None):
+        try:
+            vals = prob.get_val(f"{traj}.{phase}.timeseries.{var_name}",
+                                units=units,
+                                indices=indices,)
+
+        except KeyError:
+            try:
+                vals = prob.get_val(f"{traj}.{phase}.states:{var_name}",
+                                    units=units,
+                                    indices=indices,)
+            except KeyError:
+                try:
+                    vals = prob.get_val(f"{traj}.{phase}.timeseries.states:{var_name}",
+                                        units=units,
+                                        indices=indices,)
+                except KeyError:
+                    try:
+                        vals = prob.get_val(f"{traj}.{phase}.{var_name}",
+                                            units=units,
+                                            indices=indices,)
+                    # 2DOF breguet range cruise uses time integration to track mass
+                    except TypeError:
+                        vals = prob.get_val(f"{traj}.{phase}.timeseries.time",
+                                            units=units,
+                                            indices=indices,)
+                    except KeyError:
+                        vals = None
+
+        return vals
+
+    def _get_phase_diff(traj, phase, var_name, units, indices=[0, -1]):
+        vals = _get_phase_value(traj, phase, var_name, units, indices)
+
+        if vals is not None:
+            diff = vals[-1]-vals[0]
+            if isinstance(diff, np.ndarray):
+                diff = diff[0]
+            return diff
+        else:
+            return None
+
+    reports_folder = Path(prob.get_reports_dir())
+    report_file = reports_folder / 'mission_summary.md'
+
+    # read per-phase data from trajectory
+    data = {}
+    for idx, phase in enumerate(prob.phase_info):
+        # TODO for traj in trajectories, currently assuming single one named "traj"
+        # TODO delta mass and fuel consumption need to be tracked separately
+        fuel_burn = _get_phase_diff('traj', phase, 'mass', 'lbm', [-1, 0])
+        time = _get_phase_diff('traj', phase, 't', 'min')
+        range = _get_phase_diff('traj', phase, 'range', 'nmi')
+        # both "distance" and "range" currently exist to track ground distance, logic
+        # here accounts for that
+        if range is None:
+            range = _get_phase_diff('traj', phase, 'distance', 'nmi')
```

Comment:
Do both distance and range still exist? I thought this PR resolved that by having just distance: https://github.com/OpenMDAO/Aviary/pull/73